internal/metrics/dashboards.go

```diff
@@ -2,35 +2,30 @@ package metrics
 
 import (
 	"context"
+	"fmt"
 	"path"
 
 	log "github.com/ViaQ/logerr/v2/log/static"
-	logging "github.com/openshift/cluster-logging-operator/apis/logging/v1"
 	"github.com/openshift/cluster-logging-operator/internal/reconcile"
 	"github.com/openshift/cluster-logging-operator/internal/runtime"
 	"github.com/openshift/cluster-logging-operator/internal/utils"
 	"github.com/openshift/cluster-logging-operator/internal/utils/comparators"
 	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 const (
-	FluentdDashboardFile = "dashboards/fluentd/openshift-logging-dashboard.json"
-	VectorDashboardFile  = "dashboards/vector/openshift-logging-dashboard.json"
-	DashboardName        = "grafana-dashboard-cluster-logging"
-	DashboardNS          = "openshift-config-managed"
-	DashboardFileName    = "openshift-logging.json"
-	DashboardHashName    = "contentHash"
+	DashboardFile     = "dashboards/openshift-logging-dashboard.json"
+	DashboardName     = "grafana-dashboard-cluster-logging"
+	DashboardNS       = "openshift-config-managed"
+	DashboardFileName = "openshift-logging.json"
+	DashboardHashName = "contentHash"
 )
 
-func newDashboardConfigMap(collectionType logging.LogCollectionType) *corev1.ConfigMap {
-	var spec string
-	if collectionType == logging.LogCollectionTypeFluentd {
-		spec = string(utils.GetFileContents(path.Join(utils.GetShareDir(), FluentdDashboardFile)))
-	} else if collectionType == logging.LogCollectionTypeVector {
-		spec = string(utils.GetFileContents(path.Join(utils.GetShareDir(), VectorDashboardFile)))
-	}
+func newDashboardConfigMap() *corev1.ConfigMap {
+	spec := string(utils.GetFileContents(path.Join(utils.GetShareDir(), DashboardFile)))
```

Comment:
Shouldn't we be able to get rid of the ShareDir here and replace it with the "embedd" pattern similar to what we have done elsewhere?